packages/admin/src/pages/dashboard/index.page.tsx

```diff
@@ -40,17 +43,37 @@ const Dashboard = ({
   schemes,
   userDetails,
   oneLoginTransferErrorEnabled,
+  migrationSucceeded,
 }: InferProps<typeof getServerSideProps>) => {
   return (
     <div className="govuk-grid-row govuk-!-padding-top-7">
       <Meta title="Dashboard - Manage a grant" />
       <div className="govuk-grid-column-two-thirds govuk-!-margin-bottom-6">
-        {oneLoginTransferErrorEnabled && (
-          /* TODO: Placeholder for GAP-2013 */
-          <div>
-            <h1>PLACEHOLDER FOR ERROR BANNER</h1>
-          </div>
+        {oneLoginTransferErrorEnabled && migrationSucceeded === 'true' && (
```

Comment:
This is nitpicky but we might be better off calling this `migrationStatus` and setting it to `success` or `error` depending? It's a bit more semantically straightforward than `migrationSucceeded=false` indicating the presence of an error (vs no migration having been attempted).

Reply:
Also prob a bit more readable to name these variables in the component body:

```js
const showMigrationSuccessBanner = oneLoginTransferErrorEnabled && migrationSucceeded === 'true';
const showMigrationErrorBanner = oneLoginTransferErrorEnabled && migrationSucceeded === 'false';
```